lib/Doctrine/DBAL/Platforms/MySqlPlatform.php

```diff
@@ -1065,6 +1065,14 @@ public function getBinaryMaxLength()
         return 65535;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getMaximumNumberOfPreparedStatementPlaceholders()
+    {
+        return 65535;
```

Comment:
In non-abstract platforms, please add links to the platform documentation which the numbers are taken from,